packages/rsocket-flowable/src/Single.js

```diff
@@ -89,6 +89,12 @@ export default class Single<T> {
     });
   }
 
+  static never<U = empty>(): Single<U> {
+    return new Single(subscriber => {
+      subscriber.onSubscribe();
```

Comment:
needs at least empty object

Reply:
Yep, that's what I thought too.  But this matches `of()` and `error()` above:
https://github.com/rsocket/rsocket-js/blob/c0f461f69d43a5d9a5e60bc0408c60d8072f36e1/packages/rsocket-flowable/src/Single.js#L78-L90

Or I'm happy to change all of them if you like?

Reply:
Oh. I see. Do not worry. I will address that later. Need to finish TCK first  to see what is missing in our impl

Reply:
Looks like the `subscriber` param in all of these is an `IFutureSubject`:
https://github.com/rsocket/rsocket-js/blob/c0f461f69d43a5d9a5e60bc0408c60d8072f36e1/packages/rsocket-flowable/src/Single.js#L23

whose `onSubscribe()` param is optional:

https://github.com/rsocket/rsocket-js/blob/c0f461f69d43a5d9a5e60bc0408c60d8072f36e1/packages/rsocket-flowable/src/Single.js#L36-L40